nano/node/repcrawler.hpp

```diff
@@ -99,7 +96,7 @@ class rep_crawler
 	 * with high probability that the endpoint is a representative node.
 	 * @return True if the rep entry was updated with new information due to increase in weight.
 	 */
-	bool response (std::shared_ptr<nano::transport::channel> channel_a, nano::account const & rep_account_a, nano::amount const & weight_a);
+	void response (std::shared_ptr<nano::transport::channel> channel_a, std::shared_ptr<nano::vote>);
```

Comment:
The `vote` can be passed by reference (`channel` can do although this is existing code). Saves an inc/decrement of the atomic reference count.